tests/suites/test_suite_bignum_core.function

```diff
@@ -941,3 +941,51 @@ exit:
     mbedtls_mpi_free( &RR_REF );
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void mpi_core_ct_uint_table_lookup( int bitlen, int window_size )
+{
+    size_t limbs = BITS_TO_LIMBS( bitlen );
+    size_t count = ( (size_t) 1 ) << window_size;
+
+    mbedtls_mpi_uint *table = NULL;
+    ASSERT_ALLOC( table, limbs * count );
+
+    mbedtls_mpi_uint *dest = NULL;
+    ASSERT_ALLOC( dest, limbs );
+
+    /*
+     * Fill the table with a unique counter so that differences are easily
+     * detected. (And have their relationship to the index relatively non-trivial just
+     * to be sure.)
+     */
+    for( size_t i = 0; i < count * limbs; i++ )
+    {
+        table[i] = ~i - 1;
+    }
+
+    for( size_t i = 0; i < count; i++ )
+    {
+        /*
+         * We shouldn't leak anything through timing.
+         * We need to set these in every loop as we need to make the loop
+         * variable public for the loop head and comparison macros make the
+         * rest public automatically.
+         */
+        TEST_CF_SECRET( &i, sizeof( i ) );
+        TEST_CF_SECRET( dest, limbs * sizeof( mbedtls_mpi_uint ) );
+        TEST_CF_SECRET( table, count * limbs * sizeof( mbedtls_mpi_uint ) );
+
+        memset( dest, 0x00, limbs * sizeof( *dest ) );
+        mbedtls_mpi_core_ct_uint_table_lookup( dest, table, limbs, count, i );
+
+        mbedtls_mpi_uint *current = table + i * limbs;
+        ASSERT_COMPARE( dest, limbs * sizeof( *dest ),
+                        current, limbs * sizeof( *current ) );
+        TEST_CF_PUBLIC( &i, sizeof( i ) );
+    }
+
+exit:
```

Comment:
(must change) `dest` and `table` should be freed

Reply:
Yep, that is one of the reasons for the `needs: work` label.